kmall.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+`#!/usr/bin/env python
```

Comment:
Is the back quote at the beginning of this line intentional? What is this?

Reply:
Oops, not intentional.  Should I redo the pull request with changes?

Reply:
I think the pull request might get updated if you simply commit the fix again.

Reply:
OK, i see the change in the stack now.  Maybe that worked?